src/lib/components/table-editable/TableCellSelector.svelte

```diff
@@ -33,13 +47,23 @@
     );
   };
 
-  $: options = columnConfig.cellRenderer.options;
+  let tdElt: HTMLTableCellElement;
+  let tdHeight: number = undefined;
+  onMount(() => {
+    tdHeight = tdElt.getBoundingClientRect().height;
+  });
+  const ALERT_CIRCLE_SIZE_PX = 20;
+  $: alertCircleTop = tdHeight / 2 - ALERT_CIRCLE_SIZE_PX / 2;
```

Comment:
i know it's a little weird to set the positioning of this with JS by checking on the height of the container, but this approach fits with some other plans I have for the table layout. (And I guess in any case it's probably temporary while we work on our component sets)

Reply:
what's the intention here?